CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.3.1
+ - Fix deadlock that could occur in certain odd situations. If you noticed this plugin stalling this should fix it.
```

Comment:
@jsvd well, I actually couldn't repo it against a slow endpoint. I think from a user's perspective this is in the category of must upgrade. I think it makes sense to modify the message to say "All users should upgrade, even if they aren't experiencing problems". 

I don't know if the changelog is the place to go deep into code however.